tests.py

```diff
@@ -311,6 +311,28 @@ def test_import_from_qtwidgets():
     assert QPushButton.__name__ == "QPushButton", QPushButton
 
 
+def test_pyside2uic():
```

Comment:
There's no need to include test for this. We can expect the PySide team to test their own tools.